### PR TITLE
V1.0

### PR DESCRIPTION
**Pull request by dkorolev**
The first public release.

### CODE REVIEW

**Comment by dkorolev** [comment]
TO: @deathbaba 
CC: @sompylasar, @mzhurovich 

This is the 1.0 snapshot I am releasing, no shit, now.

Ah, and a code review would be nice too.

Thanks!
Dima

**Comment by dkorolev** [comment]
I hope the above minor nits mark the release complete.

Announcements:

* [Twitter](https://twitter.com/KnowSheet/status/573286719786631169)
* [Facebook](https://www.facebook.com/dima.korolev.16/posts/857838227608825)
* [Quora](http://dimakorolev.quora.com/Blog-Bricks-1-0-Release)

PTAL at this PR.

Thanks,
Dima

File: file/file.h [hunk 1]
```diff
@@ -207,7 +207,9 @@ struct FileSystem {
     }
 #else
     DIR* dir = ::opendir(directory.c_str());
-    const auto closedir_guard = MakeScopeGuard([dir]() { if (dir) ::closedir(dir); });
+    const auto closedir_guard = MakeScopeGuard([dir]() {
+      if (dir) ::closedir(dir);
```
Comment (sompylasar) [inline file/file.h]: braces for `if (dir)` ?

Comment (dkorolev) [inline file/file.h]: Good eye! Fixed.

**Pull request merged**